rviz_default_plugins/CMakeLists.txt

```
@@ -750,6 +752,23 @@ if(BUILD_TESTING)
     )
   endif()
 
+  ament_add_gtest(effort_display_visual_test
+    test/rviz_default_plugins/displays/effort/effort_display_visual_test.cpp
+    test/rviz_default_plugins/publishers/effort_publisher.hpp
+    test/rviz_default_plugins/page_objects/effort_display_page_object.cpp
+    ${TEST_FIXTURE_SOURCES}
+    ${SKIP_VISUAL_TESTS}
+    TIMEOUT 180)
+  if(TARGET effort_display_visual_test)
+    target_include_directories(effort_display_visual_test PUBLIC
+      test/page_objects
+      ${TEST_INCLUDE_DIRS}
+      ${rviz_visual_testing_framework_INCLUDE_DIRS})
+    target_link_libraries(effort_display_visual_test
+      ${TEST_LINK_LIBRARIES}
+      rviz_visual_testing_framework::rviz_visual_testing_framework)
+  endif()
```

With the latest changes to the CMakeLists.txt, some of this isn't needed anymore:

```suggestion
  ament_add_gtest(effort_display_visual_test
    test/rviz_default_plugins/displays/effort/effort_display_visual_test.cpp
    test/rviz_default_plugins/publishers/effort_publisher.hpp
    test/rviz_default_plugins/page_objects/effort_display_page_object.cpp
    ${SKIP_VISUAL_TESTS}
    TIMEOUT 180)
  if(TARGET effort_display_visual_test)
    target_include_directories(effort_display_visual_test PRIVATE test)
    target_link_libraries(effort_display_visual_test
      rviz_visual_testing_framework::rviz_visual_testing_framework
      rclcpp::rclcpp
      ${sensor_msgs_TARGETS}
    )
  endif()
```